### ISSUE TITLE
Keyboard navigation doesn't work with disabled inputs on DataTable with editMode="cell"

### ISSUE THREAD

**Issue by mertsincan** [primefaces/primefaces#2656]
Test code;
```
<p:column headerText="Year">
   <p:cellEditor>
       <f:facet name="output"><h:outputText value="#{car.year}" /></f:facet>
        <f:facet name="input"><p:inputText value="#{car.year}" style="width:96%" label="Year" disabled="true"/></f:facet>
   </p:cellEditor>
</p:column>
```
